app/main/page.tsx

```diff
@@ -68,17 +70,11 @@ function App() {
         </div>
       </div>
 
-      <div className="button-rows">
-        {state.waveHands.map((waveHand, index) => (
-          <button
-            key={index}
-            className={`wave-hand-button ${state.selectedWaveHand === index ? 'selected' : ''}`}
-            onClick={() => handleWaveHandsClick(index)}
-          >
-            {waveHand}
-          </button>
-        ))}
-      </div>
+      <WaveHandPicker
+        currentWaveHands={savedWaveHandButtons}
+        selectedWaveHand={-1}
```

Comment:
If we're not using state to keep track of the `selectedWaveHand`, I'd be inclined to remove it from the API of `WaveHandPicker` and only add it when/if we actually start using it.

Reply:
Removed.